### ISSUE TITLE
Failing to fetch gem directly from github for branch rails-4-2

### ISSUE THREAD

**Issue by santumahapatra** [activeadmin/inherited_resources#392]
**Failing Case:**

``` bash
gem 'inherited_resources', github: 'josevalim/inherited_resources', branch: 'rails-4-2'
```

Adding 'inherited_resources' with branch 'rails-4-2' fails with message 

``` bash
Resolving dependencies...
Could not find gem 'inherited_resources (~> 1.6) ruby', which is required by gem 'activeadmin (>= 0) ruby', in any of the
sources.
```

**Passing case:**

``` bash
gem 'inherited_resources', github: 'josevalim/inherited_resources'
```

This one works. 

**Possible solution:**

Merge the commit 9b1386f0effe9510e132915fa329f353f35db41c into branch 'rails-4-2'.

**Comment by santumahapatra** [comment]
Found the issue while trying to work with ActiveAdmin with Rails 4.2 which recommends using the inherited-resources 'rails-4-2' branch. 

This issue might affect new users of ActiveAdmin.

**Comment by rafaelfranca** [comment]
You don't need to use rails-4-2 branch anymore. It is merged at master and it was already released as 1.6.0. Use the released gem.
